.ruby-version

```diff
@@ -0,0 +1 @@
+3.0.2
```

Comment:
can you change ruby version to 2.7.1? this is the version that has in 0.25 version.